<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BranchesStatusWidget</class>
 <widget class="QWidget" name="BranchesStatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Branches</string>
  </property>
  <property name="windowIcon">
   <iconset theme="git-branch">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Reference branch:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxReferenceBranch"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PushButton" name="pushButtonNew">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset theme="document-new">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="pushButtonCheckout">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Checkout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="pushButtonRemoveSelected">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="pushButtonBrowse">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="pushButtonDiff">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Diff</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>core/pushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxReferenceBranch</tabstop>
  <tabstop>pushButtonNew</tabstop>
  <tabstop>pushButtonCheckout</tabstop>
  <tabstop>pushButtonRemoveSelected</tabstop>
  <tabstop>pushButtonBrowse</tabstop>
  <tabstop>pushButtonDiff</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
